deploy-cloudgov-docker.sh

```diff
@@ -119,34 +144,18 @@ if [ "$1" = "setup" ] ; then  echo
 	    sleep 2
 	  fi
 	fi
-
-	# set up backend
-	if cf app $CGHOSTNAME_BACKEND >/dev/null 2>&1 ; then
-		echo $CGHOSTNAME_BACKEND app already set up
-	else
-	    update_backend
-		cf bind-service $CGHOSTNAME_BACKEND tdp-db
-		cf restage $CGHOSTNAME_BACKEND
-	fi
-
-	# set up frontend
-	if cf app $CGHOSTNAME_FRONTEND >/dev/null 2>&1 ; then
-		echo $CGHOSTNAME_FRONTEND app already set up
-	else
-	    update_frontend
-	fi
```

Comment:
My working theory is that the `else` conditionals here are never executed, so these lines of code don't do anything besides say "app already set up" no matter what, and should be removed.

Here is the documentation for `cf app`: http://cli.cloudfoundry.org/en-US/v6/app.html.

The `cf app` command outputs strings that describe the health and status of a given app, but the documentation doesn't mention anything about different exit values.

My assumption is that `cf app ...` exits successfully with `0` every time, and since zero is truthy in the shell, the `else` statement in the conditionals never runs.

The actions after the `else` statement (which are never run), such as re-staging the application, binding services, and pushing new code to Cloud.gov, can all be done manually by developers if apps are crashing or otherwise not working.